core/sdk-core/pom.xml

```diff
@@ -248,6 +248,14 @@
                     </archive>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>9</source>
+                    <target>9</target>
```

Comment:
Wouldn't this cause issues running this jar on JDK 8?

Reply:
This is configuration that needed to be removed. It's a good catch because it probably does violate JRE8 compatibility inside the sdk-core module. I tested invocations with a new dummy module. I'm going to delete this chunk as no specific compiler plugin configuration should be needed here.